src/Components/Bookslist.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Msgtitle from '../Containers/msgtitle';
+import Booksform from '../Containers/booksform';
+import Book from './Ennumerated_Components/Book';
+
+function Bookslist(props) {
+  const { bookstore } = props;
+
+  setTimeout(() => {
+    document.querySelectorAll('.notshown').forEach(ele => {
+      ele.classList.remove('notshown');
+    });
+  }, 1);
+
+  return (
+    <div className="Menu">
+      <Msgtitle />
+      <Booksform />
+      {bookstore.map(ele => <Book key={ele.id} book={ele} />)}
+    </div>
+  );
+}
+
+const mapStateToProps = state => ({ bookstore: state.book });
```

Comment:
Please move components that connect to the Redux store to the containers folder, as you did with booksform.js.